contrib/perf/get-pr-info.py

```
@@ -54,19 +55,45 @@ def get_prs_from_event_file(event_path: Path) -> List[int]:
 
-    return pr_numbers
+    head_branch = workflow_run.get('head_branch')
+    if not head_branch:
+        return []
+
+    print(f"Searching for PRs by head_branch: {head_branch}", file=sys.stderr)
+
+    # Use gh CLI to search for PRs by head branch
+    try:
+        result = subprocess.run(
+            ['gh', 'pr', 'list', '--head', head_branch, '--json', 'number', '--jq', '.[].number'],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)